### ISSUE TITLE
Изменить вкладки для заявок в разделе заявок

### ISSUE THREAD

**Issue by kushedow** [kushedow/imperia#22]
По умолчанию все еще открыта вкладка новых

![image](https://user-images.githubusercontent.com/7995387/29488235-7676cb2e-850f-11e7-9769-22928f043e15.png)

**Comment by nismangulov** [comment]
@kushedow а какая должна быть открыта?

**Comment by kushedow** [comment]
Сейчас открыта вкладка новых, она и должны быть открыта

**Comment by nismangulov** [comment]
@kushedow то есть здесь все готово и можно закрывать ишью?

**Comment by kushedow** [comment]
Нет, сравни пожалуйста список вкладок на картинке и сейчас в системе )

**Comment by kipkaev55** [comment]
можно тогда комментарий к каждой группе заявок, как понимать что заявка относится к той или иной группе (т.е. это будет просто статус у заявки, как сейчас, либо какие-то свойства должны этому способствовать), как-нибудь списком или в табличке можно это оформить?
например:
1) все - отображаются абсолютно все заявки
2) новая - значит ученик открыл и начал проходить тест, либо прошёл его и оставил данные
3) ... 
ну или ещё как-нибудь

**Comment by kushedow** [comment]
Да, отлично.

Добавляем новые статусы:

– новая
– обработана
– распределена в группу
– посетил первое
– записался и оплатил
– архив

**Comment by kipkaev55** [comment]
можно проверять
